sefaria/helper/schema.py

```diff
@@ -338,17 +342,17 @@ def refresh_version_state(title):
     VersionState(title, {"flags": flags})
 
 
-def change_node_title(snode, old_title, lang, new_title):
+def change_node_title(snode, old_title, lang, new_title, ignore_cascade=False):
     """
     Changes the title of snode specified by old_title and lang, to new_title.
     If the title changing is the primary english title, cascades to all of the impacted objects
     :param snode:
     :param old_title:
     :param lang:
     :param new_title:
+    :param ignore_cascade:
     :return:
     """
-
     def rewriter(string):
```

Comment:
Really what we want here is a refactor that makes rewriter param a callback thats passed in, instead of hard coded to one piece of logic?